OmarEspinosaCastro/30-Agosto/JavaSort.java

```diff
@@ -0,0 +1,23 @@
+
+package collection;
+
+public class JavaSort {
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        int numberStudent  = Integer.parseInt(in.nextLine());
+        ArrayList<Student> studentList = new ArrayList<Student>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices